packages/widget_toolkit_pin/lib/src/lib_pin_code_with_biometrics/ui_components/pin_code_biometric_key_with_auto_submit.dart

```diff
@@ -1,11 +1,10 @@
 import 'package:flutter/material.dart';
-import 'auto_submit_widget.dart';
 import 'pin_code_key.dart';
 
 class PinCodeBiometricKeyWithAutoSubmit extends StatefulWidget {
   const PinCodeBiometricKeyWithAutoSubmit({
```

Comment:
The name of this widget must be `PinCodeBiometricsKey`. It is just a button without any auto submit logic inside it. This logic must be inside the bloc.